holoviews/operation/stats.py

```diff
@@ -16,7 +16,7 @@ def _kde_support(bin_range, bw, gridsize, cut, clip):
     if clip[0] is not None and np.isfinite(clip[0]):
         kmin = max(kmin, clip[0])
     if clip[1] is not None and np.isfinite(clip[1]):
-        kmax = max(kmax, clip[1])
+        kmax = min(kmax, clip[1])
```

Comment:
Given this function existed before this PR, what might have been affected by this bug? (if anything?)

Reply:
``Distribution`` elements would have been clipped incorrectly but I think clipping is disabled by default so not all that much would have changed.